scripts/xtensa-build-all.sh

```diff
@@ -248,7 +253,12 @@ do
 		-DCMAKE_VERBOSE_MAKEFILE=ON \
 		..
 
-	make ${PLATFORM}_defconfig
+	if [[ "x$MAKE_MENUCONFIG" == "xyes" ]]
+	then
+		make menuconfig
```

Comment:
Should we also make ${PLATFORM}_defconfig first when we want to manually config? So we can select/unselect some options only instead of select all we need for the platform.

Reply:
@bardliao are you saying we run make ${PLATFORM}_defconfig to generate the .config and then invoke make menuconfig ? 

What I noticed was that even without doing this, all the component options are selected by default when make menuconfig is invoked

Reply:
@ranj063 @bardliao 
Actually @bardliao is right. We should run defconfig then menuconfig.
The configs that you see when launching menuconfig, are from generated/.config (if for example it's not clear run and you built some platform before in the same folder). If it's clean build then defaults are based on Kconfig `default` settings that are generic and are not defaults for some specific platform